sdk/vc-data/src/data/education/v1.test.ts

```diff
@@ -0,0 +1,100 @@
+import { VCEducationPersonV1, VCSEducationPersonV1, getVCEducationPersonV1Context } from './v1'
+import { expandVC } from '../../testUtil.test'
+
+describe('VCEducationPersonV1', () => {
+  it('expands correctly', async () => {
+    expect.assertions(1)
+
+    const expanded = await expandVC<VCEducationPersonV1, VCSEducationPersonV1>({
+      type: 'EducationCredentialPersonV1',
+      data: {
+        '@type': ['Person', 'PersonE', 'EducationPerson'],
+        name: 'Bob Belcher',
+        hasCredential: {
+          '@type': 'EducationalOcupationalCredential',
+          credentialCategory: 'degree',
+          educationalLevel: 'Bachelor of Science',
+          recognizedBy: {
+            '@type': ['Organization', 'OrganizationE'],
+            name: 'University of New York',
+          },
+          dateCreated: '2020-12-07',
+          url: 'https://www.university.edu/credential/credentialId',
+        },
+      },
+      context: getVCEducationPersonV1Context(),
+    })
+
+    expect(expanded).toMatchInlineSnapshot(`
+      Object {
+        "@id": "urn:uuid:9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d",
+        "@type": Array [
+          "https://www.w3.org/2018/credentials#VerifiableCredential",
+          "https://schema.affinity-project.org/EducationCredentialPersonV1",
```

Comment:
if this is easily configurable, shall we switch to affinidi?

Reply:
it might lead to some inconsistencies in VC validation to switch so I'd prefer not to at the moment. perhaps when we move the schema management elsewhere and make these URLs live